Libraries/Animated/__tests__/Animated-web-test.js

```diff
@@ -0,0 +1,187 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @oncall react_native
+ */
+
+let Animated = require('../Animated').default;
+let AnimatedProps = require('../nodes/AnimatedProps').default;
+
+jest.mock('../../BatchedBridge/NativeModules', () => ({
+  NativeAnimatedModule: {},
+  PlatformConstants: {
+    getConstants() {
+      return {};
+    },
+  },
+}));
```

Comment:
These modules don't exist on web (even as mocks), so this mock might be setting up unrealistic expectations if the tests fail without this.

Reply:
Removed